src/libs/actions/NameValuePair.js

```diff
@@ -1,23 +0,0 @@
-import _ from 'underscore';
-import Onyx from 'react-native-onyx';
-import * as DeprecatedAPI from '../deprecatedAPI';
-
-/**
- * Sets the value for an NVP
- *
- * @param {String} name
- * @param {*} value
- * @param {String} [onyxKeyName]
- */
-function set(name, value, onyxKeyName) {
-    DeprecatedAPI.SetNameValuePair({name, value: _.isObject(value) ? JSON.stringify(value) : value});
```

Comment:
Was this really ONLY used to remove the `CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER` nvp??

Reply:
at this point, yes. It had many usages that were gradually removed over the course of the API refactors.